fiftyone/core/singletons.py

```diff
@@ -36,7 +36,11 @@ def __call__(cls, name=None, _create=True, *args, **kwargs):
             instance.__init__(name=name, _create=_create, *args, **kwargs)
             name = instance.name  # `__init__` may have changed `name`
         else:
-            instance._update_last_loaded_at()
+            try:
+                instance._update_last_loaded_at()
+            except ValueError:
+                instance._deleted = True
+                return cls.__call__(name=name, _create=False, *args, **kwargs)
```

Comment:
nit: `_create=_create` might be slightly more readable to convey that you're just re-invoking the method?